cogs/dev.py

```diff
@@ -83,6 +93,36 @@ async def sync(self, ctx: commands.Context, guilds: commands.Greedy[discord.Obje
 
         await ctx.send(f"Synced the tree to {ret}/{len(guilds)}.")
 
+    @commands.group(aliases=['ext'])
+    async def extensions(self, ctx: commands.Context):
+        """Utilities for extensions"""
+        if ctx.invoked_subcommand is None:
+            await ctx.send_help("ext")
```

Comment:
Didn't know `ctx.send_help` existed - we've been using `await available_subcommands(ctx)` for command groups. To ensure consistency for now, would prefer to use that function here.

Maybe re-working that function to include `ctx.send_help` can be done in the future (in #36)